### ISSUE TITLE
Stats command breaks on servers with many channels

### ISSUE THREAD

**Issue by tadeokondrak** [nixnest/scepter#16]
```
WARNING: `stats ` errored with `RangeError: RichEmbed field values may not exceed 1024 characters.`
```

**Comment by ldelelis** [comment]
Proposals:

1* If the first embed exceeds 1024 characters, send a second embed with the rest of the channels (the easy, ugly way out)
2* Create a pagination mechanism, to move back and forth between the channels within the same embed (the hard, pretty way out)

**Comment by tadeokondrak** [comment]
Having a pagination method in a lib/ function that any module can use could be a good solution

**Comment by ldelelis** [comment]
Partly fixed with #24, waiting for code review and bug fixes before considering this closed.
